requests_mv_integrations/request_mv_integration.py

```diff
@@ -0,0 +1,1774 @@
+            extra=response_extra
+        )
+
+        return json_response
+
+    def requests_response_json(
+        self,
+        response,
+        request_label=None,
+        raise_ex_if_not_json_response=True
+    ):
+        """Get JSON from response from requests
+
+        Args:
+            response:
+            request_label:
+
+        Returns:
+
+        """
+        json_response = None
+        response_extra = {}
+        if request_label:
+            response_extra.update({
+                'request_label': request_label
+            })
+
+        try:
+            json_response = response.json()
+            response_details_source = 'json'
+            response_content_length = len(json_response)
+
+            response_extra.update({
+                'response_details_source': response_details_source,
+                'response_content_length': response_content_length
+            })
+        except json.decoder.JSONDecodeError as json_decode_ex:
+            self.logger.error(
+                "Validate JSON Response: Failed: JSONDecodeError",
+                extra=response_extra
+            )
+
+            data = requests_toolbelt.utils.dump.dump_all(response)
+            pprint(data.decode('utf-8'))
+
+            pprint(response.text)
+
+            self.handle_json_decode_error(
+                response_decode_ex=json_decode_ex,
+                response=response,
+                response_extra=response_extra,
+                request_label=request_label,
+                request_curl=self.built_request_curl
+            )
+
+        except Exception as ex:
+            self.logger.error(
+                "Validate JSON Response: Failed: Exception",
+                extra=response_extra
+            )
+
+            pprint(response.text)
+
+            self.handle_json_decode_error(
+                response_decode_ex=ex,
+                response=response,
+                response_extra=response_extra,
+                request_label=request_label,
+                request_curl=self.built_request_curl
+            )
+
+        if json_response is None:
+            if raise_ex_if_not_json_response:
+                self.logger.error(
+                    "Validate JSON Response: Failed: None",
+                    extra=response_extra
+                )
+
+                raise TuneRequestModuleError(
+                    error_message="Validate JSON Response: Failed: None",
+                    error_request_curl=self.built_request_curl,
+                    exit_code=TuneIntegrationExitCode.MOD_ERR_SOFTWARE
+                )
+            else:
+                self.logger.warning(
+                    "Validate JSON Response: None",
+                    extra=response_extra
+                )
+        else:
+            self.logger.debug(
+                "Validate JSON Response: Valid",
+                extra=response_extra
+            )
+
+        return json_response
+
+    @staticmethod
```

Comment:
for staticmethods these are good candidates to free from the class and "shift to the left"

In cases where we need the logger modify the function signature to accept it.